src/twisted/names/authority.py

```diff
@@ -34,28 +44,14 @@ def getSerial(filename = '/tmp/twisted-names.serial'):
         os.umask(o)
 
     with open(filename, 'r') as serialFile:
-        lastSerial, ID = serialFile.readline().split()
+        lastSerial, id = serialFile.readline().split()
```

Comment:
`id` is a builtin, which is why this was previously capitalized :).

Perhaps `zoneID`?